### DISCUSSION TITLE
Why is there no Exclude Location for Cache (it's only avail for Indexer)?

### DISCUSSION THREAD

**Discussion by amiga-500** [Xanashi/Icaros#32] [Feature Requests]
So in Icaros under Cache there is a "Locations" tab so naturally one would think by adding a folder path to "Excluded Locations" it will not cache that directory when one dynamically roams their system and happens to go to that excluded folder. Wrong!  

This "Excluded Locations" only gets considered when the Indexer runs (ie. when u click Build Cache).

For the next version can someone just make it so entries in Locations will ALSO be considered when caching as the user navigates, not just when i manually click "Rebuild Cache"?

**Comment by Xanashi** [comment]
Hi amiga-500,

Thank you for you feedback! 

I would definitely had made Icaros respect the excluded locations during dynamic caching, if it were possible, 
but unfortunately it is not.

The reason is that during thumbnailing in Windows Explorer, the only information about the file being thumbnailed,
that Icaros gets from Windows, is the file's data (i.e. a file stream). Icaros doesn't actually know the file's location.

So, due to the above limitation, the settings on the Locations tab is only applying to indexer.

**Reply by amiga-500** [reply]
Ahhhh...much appreciated for the explanation.  Thanks very much for a fantastic product.

**Reply by Xanashi** [reply]
Thank you so much for your feedback and the kind words. ❤️  
Don't hesitate to post about any bugs or feature request you have for Icaros.
